packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -396,6 +518,25 @@ abstract class ImageStreamCompleter with Diagnosticable {
     }
   }
 
+  /// Adds a listener callback that is called whenever a new concrete [ImageInfo]
+  /// object is available or an error is reported, if there are any listeners
```

Comment:
the second part of the first sentence is confusing

Reply:
Rephrasing to hopefully help.